src/Components/Articles.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react'
+import ArticleCard from './ArticleCard';
+import './styling/articleHomePage.css'
+
+function Articles ({currArticles, setCurrArticles}) {
+    const [isLoading, setIsLoading] = useState(true)
+    function getArticles () {
+        setIsLoading(true);
+        return axios.get('https://nc-news-94l5.onrender.com/api/articles').then(({data})=> {
+            setCurrArticles(data.articles.slice(0,10))
+            setIsLoading(false)
+        })
+    }
+    useEffect(()=> {
+        getArticles()
+    }, []);
```

Comment:
Your getArticles and useEffect needs a bit of rethinking to make sure that the right thing is being handled in the right place. In getArticles, the only thing you need to do there is ensure the data itself is being returned.

All of the state-related stuff is the responsibility of the useEffect that lives in the component.

Also, it's idea to keep getArticles in it's own utils file (that we'd usually call api.js) which allows you to require it in and streamline this component file quite a bit